### PR TITLE
Creating models

### LINKED ISSUE #3: Create Models
Create Base, Profile, User, Contact and Address models

### LINKED ISSUE #14: Create BaseModel
Create a Base model having the following properties
int Id

### LINKED ISSUE #15: Create Profile Model
Create a Profile model having the following ​properties
string Name
string Stack
string Skills

### LINKED ISSUE #16: Create Adress Model
Create a Adress model having the follow properties
string Street
int Number
string Complement
string City 
string State

### LINKED ISSUE #17: Create Contact Model
Create Contact model having the follow properties
string Phone
string Email 
string LinkedIn
string GitHub

### LINKED ISSUE #18: Create Developer Model
Create Developer model having the follow properties
User type User 
Profile type Profile 
Contact type Contact 
Address type Address

### LINKED ISSUE #19: Create User Model
Create a User model having the following ​properties
string Username
string Password

### PR DESCRIPTION
**Pull request by caiomateusreis**
#3
#17 
#15 
#14
#16
#19

### CODE REVIEW

**Review by sr-adilson — APPROVED** [review]
Alright my friend!

**Review by viniVendrame — APPROVED** [review]
That's it Caio

File: Developers/Data/Developer.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data
+{
+    class Developer : Base
```
Comment (felipe-hofelmann) [inline Developers/Data/Developer.cs]: There was a public missing in the class

**Review by felipe-hofelmann — CHANGES_REQUESTED** [review]
There was a public missing in the class

**Review by felipe-hofelmann — APPROVED** [review]
Now yes

**Pull request merged**